### PR TITLE
Redesign Home Interaction

### PR DESCRIPTION
**Pull request by garrettmflynn**
This PR implements recommendations made by @oruebel (in user test) and @rly (#531) to improve the user experience of interacting with the GUIDE home screen. 

In particular, it swaps the original home screen with the Conversions page so that users can quickly engage with the main feature of the GUIDE: designing conversion pipelines.

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
Attempting to run a fresh tutorial pipeline on this PR results in my backend server crashing and restarting before it can even launch the pipeline; do I need to do something special when testing this branch?

**Comment by garrettmflynn** [comment]
Changing the conversion page path messed up the progress `resume` function. I've updated this now.

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
Love the consistency of the new workflow names!

All works for me

**Comment by CodyCBakerPhD** [comment]
Storybook has some problems that start on the first couple of pipeline pages though

**Comment by garrettmflynn** [comment]
Good catch! Similar issue, just fixed

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
> Good catch! Similar issue, just fixed

Most still seem to be empty: https://www.chromatic.com/test?appId=640a4d93e34604f275368983&id=659f074e4699c6ad0c249ece

File: src/renderer/src/stories/pages/guided-mode/GuidedHome.js [hunk 1]
```diff
@@ -183,14 +178,24 @@ export class GuidedHomePage extends Page {
...
+                    <small
+                        >Don't know where to go next? Learn more about the NWB GUIDE
+                        <a
+                            href=""
+                            @click="${(clickEvent) => {
+                                clickEvent.preventDefault();
+                                this.to("docs");
+                            }}"
+                            >here</a
+                        >.</small
```
[the remark this answers is not in the document]
Comment (garrettmflynn) [reply src/renderer/src/stories/pages/guided-mode/GuidedHome.js]: Looks good to me!

**Comment by garrettmflynn** [comment]
Got it. That one had an inconsistent format for the updated line so it was missed on my first pass. Thanks!

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**